scripts/dev/custom_check.sh

```diff
@@ -1,25 +1,38 @@
 #!/bin/bash
 
-REPO_ROOT=$1
+if test "$#" -eq 1; then
```

Comment:
Try to automatically apply the repo dir if you don't pass an argument to the custom check manager script.  I quite often just run `./scripts/dev/custom_check.sh` when I'm reviewing PRs and it complains because it expects a command line arg.  This just makes it optional.  I don't think anyone else out there will ever notice.